packages/all/package.json

```diff
@@ -24,15 +24,15 @@
     "clean": "rm -Rf ./node_modules ./lib/* ./meta/bundlesize/* ./meta/coverage/*",
     "prepare": "npm run build",
     "prebuild": "npm run clean",
-    "build": "../../node_modules/.bin/tsc -p ./config",
-    "postbuild": "../../node_modules/.bin/rollup -c ./config/rollup.config.js",
+    "build": "npx tsc -p ./config",
```

Comment:
We actually recently moved away from using `npx` with scripts in https://github.com/apollographql/react-apollo/pull/2892/commits/cda6b37f81ef19926df2eede7d4e7c623a869c7e. @benjamn any objections to moving back?

Reply:
I was just trying to fix the build on Windows, the error was ```.. is not a command```. I'm open to any solution that lets me run the build/tests in my environment :)

Reply:
@matthargett I'm going to change this around a bit, as `npx` can't use a parent's `node_modules` (it can only use a `node_modules` in the current package or global location). This means each child package `npx` call ends up re-installing the referenced packages first, since all of our devDeps are in the root of the monorepo. As a quick workaround to this, we'll add the utilities we're referring to with `npx` as devDeps in each child package. Not ideal, but we're only talking about 4 deps, and Renovate will help us keep them up to date.